packages/igniter-test-helpers/lib/after-microtasks.ts

```diff
@@ -1,23 +1,36 @@
-import Ember from 'ember';
-import { manager } from 'igniter/microtask-manager';
+/*
+ * WAT. There is some stuff in here that is Ember dependent
```

Comment:
I worked pretty hard to find a way to integrate igniter into existing Ember applications. Turns out, it's not possible without completely swapping out the backburner module.  Some of the Ember stuff is left over from that, but this particular instance is because of the use of ember-qunit, ember-cli-qunit, and RSVP.  Unfortunately RSVP flushes within backburner's action queue until Igniter lands, so if Ember + Backburner are in the same environment as Igniter any microtask based tests would fail.  Crossing fingers that this new build setup eliminates that ;)